docs/source/getting-started/setting-up-hypervisor-driver-plugin.adoc

```diff
@@ -152,5 +151,36 @@ $ sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve
 $ sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
 ----
 
-For more information, refer to the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
+For more information, see the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
 ====
+
+== Hyper-V
```

Comment:
@gbraad @Preeticp IMO this does not fit here. May be we need to find a new home for this.